lib/Driver/ToolChains.cpp

```diff
@@ -1115,8 +1115,10 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   // Add the linker script that coalesces protocol conformance sections.
   Arguments.push_back("-Xlinker");
   Arguments.push_back("-T");
-  Arguments.push_back(
-      context.Args.MakeArgString(Twine(RuntimeLibPath) + "/x86_64/swift.ld"));
+    
+  // FIXME: This should also query the abi type (i.e. gnueabihf)
```

Comment:
I think you’ll need to update the location where the stdlib is built into, otherwise the paths won’t match.  I tried this last night.

> On Dec 14, 2015, at 7:25 AM, Brian Gesiak <notifications@github.com> wrote:
> 
> In lib/Driver/ToolChains.cpp <https://github.com/apple/swift/pull/439#discussion_r47509595>:
> 
> > @@ -1115,8 +1115,10 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
> >    // Add the linker script that coalesces protocol conformance sections.
> >    Arguments.push_back("-Xlinker");
> >    Arguments.push_back("-T");
> > -  Arguments.push_back(
> > -      context.Args.MakeArgString(Twine(RuntimeLibPath) + "/x86_64/swift.ld"));
> > +    
> > +  // FIXME: This should also query the abi type (i.e. gnueabihf)
> Awesome. I'll send a pull request; there's also a test that needs updating.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/apple/swift/pull/439/files#r47509595>.
>